ui/src/components/appView/AppOverview.js

```diff
@@ -27,7 +27,7 @@ const AppOverview = ({ className, app }) => {
 AppOverview.propTypes = {
   className: PropTypes.string.isRequired,
   app: PropTypes.shape({
-    appId: PropTypes.string.isRequired
+    appId: PropTypes.string
```

Comment:
good spot :+1:

Reply:
Yeah, if the `app` is required then this will automatically be required.